x/concentrated-liquidity/swaps_test.go

```diff
@@ -0,0 +1,162 @@
+package concentrated_liquidity_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/internal/math"
+	"github.com/osmosis-labs/osmosis/v13/x/gamm/types"
+	swaproutertypes "github.com/osmosis-labs/osmosis/v13/x/swaprouter/types"
+)
+
+var _ = suite.TestingSuite(nil)
+
+func (s *KeeperTestSuite) TestCLPoolSimpleSwapExactAmountIn() {
+	currPrice := sdk.NewDec(5000)
+	currSqrtPrice, err := currPrice.ApproxSqrt() // 70.710678118654752440
+	s.Require().NoError(err)
+	currTick := math.PriceToTick(currPrice) // 85176
+	lowerPrice := sdk.NewDec(4545)
+	lowerTick := math.PriceToTick(lowerPrice) // 84222
+	upperPrice := sdk.NewDec(5500)
+	upperTick := math.PriceToTick(upperPrice) // 86129
+
+	defaultAmt0 := sdk.NewInt(1000000)
+	defaultAmt1 := sdk.NewInt(5000000000)
+
+	swapFee := sdk.ZeroDec()
+
+	type param struct {
+		tokenIn           sdk.Coin
+		tokenOutDenom     string
+		tokenOutMinAmount sdk.Int
+		expectedTokenOut  sdk.Int
+	}
+
+	tests := []struct {
+		name      string
+		param     param
+		expectErr bool
+	}{
+		{
+			name: "Proper swap foo > bar",
+			param: param{
+				tokenIn:           sdk.NewCoin("foo", sdk.NewInt(42000000)),
+				tokenOutDenom:     "bar",
+				tokenOutMinAmount: sdk.NewInt(1),
+				expectedTokenOut:  sdk.NewInt(8396),
+			},
+		},
+		{
+			name: "Proper swap bar > foo",
+			param: param{
+				tokenIn:           sdk.NewCoin("bar", sdk.NewInt(13370)),
+				tokenOutDenom:     "foo",
+				tokenOutMinAmount: sdk.NewInt(1),
+				expectedTokenOut:  sdk.NewInt(66808387),
+			},
+		},
+		{
+			name: "out is lesser than min amount",
+			param: param{
+				tokenIn:           sdk.NewCoin("foo", sdk.NewInt(42000000)),
```

Comment:
nit: seems that a  lot of the same values are reused throughout the test - `42000000`, `8397` etc. Consider initializing constants with these values and reusing them where applicable

Reply:
I kind of feel displaying the number itself, despite being repeated, is the best way. If it was instead something like "baseCaseEthIn" and "baseCaseUSDCOut", we would just need to look at what that is anyway to confirm the math is right, right? Please lmk if I am misunderstanding, I very well could be

Reply:
Fair, that makes sense! Let's keep it